packages/mobile/capacitor-sample.config.ts

```diff
@@ -10,6 +10,11 @@ const config: CapacitorConfig = {
         url: 'http://XXX.XXX.XXX.XXX:8080',
         cleartext: true,
     },
+    cordova: {
+        preferences: {
+            DisableDeploy: 'true',
+        }
+    }
```

Comment:
```suggestion
        },
    },
```

The JS format check seems to be failing because of this